thyrel-api/thyrel-api/Controllers/ElementController.cs

```diff
@@ -83,6 +83,33 @@ public async Task<ActionResult<Element>> Finish(int id, [FromBody] ElementBody b
             return Ok(finishState);
         }
 
+        [HttpPatch("auto/{id}")]
+        public async Task<ActionResult<Element>> AutoFinish(int id, [FromBody] ElementBody body)
+        {
+            var player = await AuthorizationHandler.CheckAuthorization(HttpContext, _context);
+            if (player?.RoomId == null) return Unauthorized("You're not in the room.");
+
+            var elementDataProvider = new ElementDataProvider(_context);
+            var sessionDataProvider = new SessionDataProvider(_context);
+
+            var element = await elementDataProvider.GetElement(id);
+            if (element.CreatorId != player.Id) return Unauthorized("You're not the creator of this element.");
+
+            var session = await sessionDataProvider.GetSessionById(element.SessionId);
+            if (element.Step != session.ActualStep) return Unauthorized("You can't modify a previous element.");
+
+            if (element.Type == ElementType.Sentence)
+            {
+                await elementDataProvider.SetSentence(element.Id, body.Text);
+            }
+            else
+            {
+                await elementDataProvider.SetDrawing(element.Id, body.DrawImage);
+            }
+
+            return Ok(element);

```

Comment:
```suggestion
            return Ok();
```
juste ok car tu ne vas pas que ton truc return qqch